pkg/roachprod/install/cockroach.go

```diff
@@ -1294,9 +1295,13 @@ func (c *SyncedCluster) setClusterSettings(
 func (c *SyncedCluster) generateClusterSettingCmd(
 	ctx context.Context, l *logger.Logger, node Node, virtualCluster string,
 ) (string, error) {
-	if config.CockroachDevLicense == "" {
-		l.Printf("%s: COCKROACH_DEV_LICENSE unset: enterprise features will be unavailable",
-			c.Name)
+	license := config.CockroachDevLicense
+	if license == "" {
+		license, _ = (&licenseccl.License{
+			Type:              licenseccl.License_Enterprise,
+			Environment:       licenseccl.Development,
+			ValidUntilUnixSec: timeutil.Now().AddDate(0, 1, 0).Unix(),
+		}).Encode()
```

Comment:
We should still log (at INFO level) that a fresh license was generated, on the off chance that the user was expecting `config.CockroachDevLicense` to be non-empty.